### PR TITLE
[tracing] Update APM Trace Context Propagation pages

### PR DESCRIPTION
**Pull request by zacharycmontoya**
### What does this PR do? What is the motivation?
This re-organizes the Trace Context Propagation pages to more clearly explain what the feature does, what propagators we support, and reorganize the text-heavy language-specific pages into more readable sections.

### Merge instructions
- [ ] Please merge after reviewing

### Additional notes
Preview links:
- Landing page: https://docs-staging.datadoghq.com/zach.montoya/update-propagation/tracing/trace_collection/trace_context_propagation/
- .NET page: https://docs-staging.datadoghq.com/zach.montoya/update-propagation/tracing/trace_collection/trace_context_propagation/dotnet/

For reference, the links from the prod site are:
- Landing page: https://docs.datadoghq.com/tracing/trace_collection/trace_context_propagation/
- .NET page: https://docs.datadoghq.com/tracing/trace_collection/trace_context_propagation/dotnet/

### CODE REVIEW

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 1]
```diff
@@ -16,17 +16,276 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-W3C Trace Context propagation is available for all supported languages. The default trace propagation style for all languages is `datadog,tracecontext`. Envoy and nginx proxies use the default of `tracecontext,datadog`.
+Trace Context propagation is the mechanism of passing tracing information like Trace ID, Span ID, and sampling decisions from one part of a distributed application to another. This enables all traces (and additional telemetry) in a request to be correlated. When automatic instrumentation is enabled, trace context propagation is handled automatically by the tracing library.
 
-For more information about each language's configuration options for trace context propagation, see the following pages:
+By default, the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will giver higher precedence to the Datadog format, whereas Envoy and NGINX proxies will give higher precedence to the W3C Trace Context format.
```
Comment (tushar-datadog) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:21]: *SDKs

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 2]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
 
-{{< partial name="apm/apm-context-propagation" >}}
+## Configuration
 
+If you need to customize the trace context propagation configuration, there are several environment variables you can use to configure the formats that are used for reading and writing distributed tracing headers. To enable a specific propagator, make sure to use the corresponding configuration value for the tracing library, as outlined in the **Language support** section.
```
Comment (tushar-datadog) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:25]: We should link to the language support section

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 3]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
 
-<br>
+<div class="alert alert-info">
+If multiple propagators are enabled, the extraction attempt is done in the specified order, and the first valid trace context is used to continue the distributed trace. If additional valid trace contexts are found, the tracing information will be recorded as individual span links.</div>
+
+### Recommended Datadog configuration
+
+`DD_TRACE_PROPAGATION_STYLE`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This may be overridden by the extract-specific or inject-specific configurations. <br>
+**Default:** `datadog,tracecontext` for language SDK's, `tracecontext,datadog` for Envoy and NGINX procies
```
Comment (tushar-datadog) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:34]: SDKs

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 4]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+
+### Recommended OpenTelemetry configuration
```
Comment (tushar-datadog) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:36]: Could you throw in a line here telling the visitor when to use the "Recommended Datadog configuration" or when to use the "Recommended OpenTelemetry configuration". I saw the ENV variables here, but did not understand when would good time to use either of them

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 5]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+
+`OTEL_PROPAGATORS`
```
Comment (tushar-datadog) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:38]: The DD variable has default values in the description section and this OTel variable doesn't. 
We can provide this link: https://opentelemetry.io/docs/languages/sdk-configuration/general/#otel_propagators

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 6]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+
+| Propagator             | Configuration Value |
+|------------------------|---------------------|
+| Datadog                | `datadog`           |
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Single][3]         | `b3 single header`  |
+|                        | `b3single`          |
+| [B3 Multi][4]          | `b3multi`           |
+|                        | `b3` (deprecated)   |
+| None                   | `none`              |
```
Comment (tushar-datadog) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:68]: Out of curiosity, what happens if you set none? Trace context propagation does not happen?

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 1 above)
Comment (dmehala) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:21]:
```suggestion
By default, the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will giver higher precedence to the Datadog format.
```

That's not true anymore.

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 3 above)
Comment (dmehala) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:34]:
```suggestion
**Default:** `datadog,tracecontext` for language SDK's.
```

Ditto

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 7]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+| Propagator             | Configuration Value |
+|------------------------|---------------------|
+| Datadog                | `datadog`           |
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Multi][4]          | `b3`                |
+|                        | `b3multi`           |
+| None                   | `none`              |
+
+### Additional configuration
+In addition to the environment variable configuration, you can also update the propagators in code by using `Datadog.configure`:
```
Comment (dmehala) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:216]:
```suggestion
In addition to the environment variable configuration, you can also update the propagators in code:
```

**Review by dmehala — APPROVED** [review]
That is a very welcome update, it is easy to read and understand now. Thank you! LGTM.

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 1 above)
Comment (mabdinur) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:21]:
```suggestion
By default, the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will give higher precedence to the Datadog format, whereas Envoy and NGINX proxies will give higher precedence to the W3C Trace Context format.
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 8]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+### Recommended Datadog configuration
+
+`DD_TRACE_PROPAGATION_STYLE`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This may be overridden by the extract-specific or inject-specific configurations. <br>
+**Default:** `datadog,tracecontext` for language SDK's, `tracecontext,datadog` for Envoy and NGINX procies
+
+### Recommended OpenTelemetry configuration
+
+`OTEL_PROPAGATORS`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This configuration takes the lowest precedence and will be ignored if any other Datadog propagation environment variable is set.
```
Comment (mabdinur) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:39]: We should link to https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 6 above)
Comment (mabdinur) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:68]: Yup. None disables distributed tracing

**Review by mabdinur — APPROVED** [review]

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 9]
```diff
@@ -16,17 +16,276 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-W3C Trace Context propagation is available for all supported languages. The default trace propagation style for all languages is `datadog,tracecontext`. Envoy and nginx proxies use the default of `tracecontext,datadog`.
+Trace Context propagation is the mechanism of passing tracing information like Trace ID, Span ID, and sampling decisions from one part of a distributed application to another. This enables all traces (and additional telemetry) in a request to be correlated. When automatic instrumentation is enabled, trace context propagation is handled automatically by the tracing library.
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:19]: Let's call out the current defaults very clearly here, on their own line with a description of "why"

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 1 above)
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:21]:
```suggestion
By default, all of the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will giver higher precedence to the Datadog format, whereas Envoy and NGINX proxies will give higher precedence to the W3C Trace Context format.
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 2 above)
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:25]: Let's make sure this answers "why" someone would change this

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 10]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
-{{< partial name="apm/apm-context-propagation" >}}
+## Configuration
 
+If you need to customize the trace context propagation configuration, there are several environment variables you can use to configure the formats that are used for reading and writing distributed tracing headers. To enable a specific propagator, make sure to use the corresponding configuration value for the tracing library, as outlined in the **Language support** section.
 
-<br>
+<div class="alert alert-info">
+If multiple propagators are enabled, the extraction attempt is done in the specified order, and the first valid trace context is used to continue the distributed trace. If additional valid trace contexts are found, the tracing information will be recorded as individual span links.</div>
+
+### Recommended Datadog configuration
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:30]:
```suggestion
### Services Instrumented with Datadog SDK
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 4 above)
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:36]:
```suggestion
### Services Instrumented with OpenTelemetry SDK
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 11]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+
+`DD_TRACE_PROPAGATION_STYLE`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This may be overridden by the extract-specific or inject-specific configurations. <br>
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:33]:
```suggestion
: Specifies propagators (in a comma-separated list) to be used for trace context propagation (extraction and injection). This may be overridden by the extract-specific or inject-specific configurations. <br>
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 12]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
 ---
 
-W3C Trace Context propagation is available for all supported languages. The default trace propagation style for all languages is `datadog,tracecontext`. Envoy and nginx proxies use the default of `tracecontext,datadog`.
+Trace Context propagation is the mechanism of passing tracing information like Trace ID, Span ID, and sampling decisions from one part of a distributed application to another. This enables all traces (and additional telemetry) in a request to be correlated. When automatic instrumentation is enabled, trace context propagation is handled automatically by the tracing library.
 
-For more information about each language's configuration options for trace context propagation, see the following pages:
+By default, the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will giver higher precedence to the Datadog format, whereas Envoy and NGINX proxies will give higher precedence to the W3C Trace Context format.
 
-{{< partial name="apm/apm-context-propagation" >}}
+## Configuration
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:23]:
```suggestion
## Customization
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 3 above)
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:34]: Move this to the top of the page along with why we chose this format

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 13]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+`DD_TRACE_PROPAGATION_STYLE_INJECT`
+: Specifies propagators (in a comma-separated list) to be used only for trace context injection. This configuration takes the highest precedence over all other configurations for configuring the injection propagators.
+
+## Language support
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+### Supported propagators
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:57]:
```suggestion
### Supported formats
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 14]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+The Datadog Java SDK maintains several propagators for passing trace context information in different formats:
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:58]:
```suggestion
The Datadog Java SDK supports the following trace context formats:
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Propagating the Trace Context
+title: Trace Context Propagation
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:2]: For some reason, going to this page puts us in the OTEL part of the docs, rather than APM. Can we fix that?
image

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 16]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+## Language support
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+### Supported propagators
+The Datadog Java SDK maintains several propagators for passing trace context information in different formats:
+
+| Propagator             | Configuration Value |
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:60]:
```suggestion
| Format             | Configuration Value |
```

Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:60]: [nit] This bit can be made common across all languages so we can shift the tabs down a bit
```
### Supported formats
The Datadog Java SDK supports the following trace context formats:

{{< tabs >}}

{{% tab "Java" %}}
...
```

File: content/en/tracing/trace_collection/trace_context_propagation/cpp.md [hunk 17]
```diff
@@ -13,77 +13,9 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-## Overview
+This page documents additional use cases of trace context propagation for the Datadog C++ SDK. For a high-level overview of trace context propagation, see [Trace Context Propagation][1].
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/cpp.md:16]: Rather than having a page here, can we put this into a collapsable section within the CPP tab of the main page? The same goes for .NET and PHP

```
{{% collapse-content title="Additional Use Cases" level="h4" %}}
Content goes here
{{% /collapse-content %}}
```

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 6 above)
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:68]: Now may be a good time to standardize on the naming here across languages. We can have our table on the main page be the intersection of all languages and then language-specific tabs can describe "deprecated" behavior

To be clear, we can standardize around the following values across all languages
`datadog, tracecontext, b3single, b3multi, none` and mark the other aliased names as deprecated

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 18]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+|------------------------|---------------------|
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:61]: Java should have XRay here

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 6 above)
Comment (bm1549) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:68]: Let's add this to our docs

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 19]
```diff
@@ -16,17 +16,276 @@ further_reading:
...
+| Datadog                | `datadog`           |
```
Comment (bm1549) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:62]: I realize we don't currently have a page that defines this format - can we add a small page describing the header values as part of this change?

**Review by bm1549 — COMMENTED** [review]
The changes here are excellent! This makes a step-difference in taking us from language-specific tracers to a Datadog SDK

I dropped a lot of nit-picky comments, but the overall contents here are incredible 🙏

File: content/en/tracing/trace_collection/trace_context_propagation/cpp.md (hunk 17 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/cpp.md:16]: Great idea, will do!

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 1 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in a9a5bb178095700f451145f880837f7da8a23338

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 3 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed SDKs in a9a5bb178095700f451145f880837f7da8a23338

Fixed Envoy/NGINX in 90fb57a997ae0d771cfae5c30b0a452342c07783

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 1 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in a9a5bb178095700f451145f880837f7da8a23338

Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in a9a5bb178095700f451145f880837f7da8a23338

Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in a9a5bb178095700f451145f880837f7da8a23338

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 7 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in 5537d5bcfd719ca17485bd10969911540378ae72

**Review by mabdinur — COMMENTED** [review]
This document many covers trace context propagation for ddtrace/otel integrations. It would also be nice to mention support for manual context propagation. In python customer's can use ``inject`` and ``extract`` methods on the [HTTPPropagator](https://ddtrace.readthedocs.io/en/stable/api.html#ddtrace.propagation.http.HTTPPropagator) class to send/receive distributed trace info. I think we can do something similar in all tracing clients.

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 14 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in ca792a35e1ce6b68fe8d312c91884e25eb1d6e46

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 16 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in ca792a35e1ce6b68fe8d312c91884e25eb1d6e46

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 13 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in ca792a35e1ce6b68fe8d312c91884e25eb1d6e46

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 18 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:61]: Fixed in a9242efd510965daa2b6d98c64a4cd575fa78de2

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 19 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Fixed in a9242efd510965daa2b6d98c64a4cd575fa78de2 

I added a section to this page with the definition of the Datadog headers

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 6 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: I've added a small section in this commit: https://github.com/DataDog/documentation/commit/a9242efd510965daa2b6d98c64a4cd575fa78de2

File: content/en/tracing/trace_collection/trace_context_propagation/cpp.md (hunk 17 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/cpp.md]: Done in the following commits:
- cc946ea56b14ea55d7053b90ba0175646535fbd6
- 80b9f4a42b0c8a92cd1ef332965613811ca4ccc8
- f13ef990ddaac9ec22a705ed55a42363d55b51d3

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 9 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:18]: Updated in 89d3d40e48235ff93a8c9f6472ff2c66a587d95e

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 2 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Updated in 89d3d40e48235ff93a8c9f6472ff2c66a587d95e

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 9 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:18]: @brett0000FF flagging because I want your review to make sure that the defaults and their "why" is clear to you/users

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 2 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: @brett0000FF flagging because I want your review to make sure that "why" behind changing the configuration is clear to you/users

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 3 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: @brett0000FF I wonder what you think on this. I've now listed the default values at the top, so I'm wondering if we can leave the `DD_TRACE_PROPAGATION_STYLE` configuration definition where it is

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 6 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: I think this is a good idea and we could standardize on `datadog, tracecontext, b3multi, none` but don't have a standardized name for the `b3single` style. We could enter this into the table and say that it's language-dependent?

@brett0000FF I wonder what you think?

Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: I've added an experimental commit to add a consolidated table, in addition to the language-specific tables: 9c42a0d3643c8a130d184bb7005763c02d98b4a6

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 16 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: I think the language-specific formats should be inside the language tab, so I'll leave this as-is. I appreciate the feedback though

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 15 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:2]: I've attempted to fix this issue in commit e02c2fe202d29f8c88ea831534a45daae200daf5

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: We don't historically have one, so I've added a new section on this page with the expected Datadog headers: 56c3afefbfe79ad5d48554f47187e620aa4fd2df

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 2 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: Added a link in aece8fb4f4281d29712db84828c2858300ef7e48 👍🏼

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 4 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: I've added this in-line to the `OTEL_PROPAGATORS` description in aece8fb4f4281d29712db84828c2858300ef7e48

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 5 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:39]: I've added a link to our now-finished https://docs.datadoghq.com/opentelemetry/interoperability/environment_variable_support page. Does that work for you?

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 8 above)
Comment (zacharycmontoya) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md]: I've added a link to our now-live https://docs.datadoghq.com/opentelemetry/interoperability/environment_variable_support page. WDYT?

**Comment by zacharycmontoya** [comment]
> This document many covers trace context propagation for ddtrace/otel integrations. It would also be nice to mention support for manual context propagation. In python customer's can use `inject` and `extract` methods on the [HTTPPropagator](https://ddtrace.readthedocs.io/en/stable/api.html#ddtrace.propagation.http.HTTPPropagator) class to send/receive distributed trace info. I think we can do something similar in all tracing clients.

Yes 100%, and we're doing that with .NET / PHP / C++ in this commit. I'm cool with adding that as a follow-up PR so we can expedite this along, but whatever format we standardize here for the manual code samples we can easily adopt for the Python one. Does that sound good to you?

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md (hunk 15 above)
Comment (brett0000FF) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:2]: This is because the menu file is pointing to the same page from two locations (and it prefers the one higher up, aka OTel). I've fixed this here b9bdc32d12d65efe939583b833e70fc74baf0563

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 20]
```diff
@@ -15,17 +15,616 @@ further_reading:
...
+When the Datadog SDK is configured with the Datadog format for extraction or injection (possibly both), the Datadog SDK interacts with the following request headers:
+
+`x-datadog-trace-id`
+: Specifies the lower 64-bits of the 128-bit trace-id, in decimal format.
+
+`x-datadog-parent-id`
+: Specifies the 64-bits span-id of the current span, in decimal format.
+
+`x-datadog-origin`
+: Specifies the Datadog service that initiated the trace, such as [Real User Monitoring][7] or [Synthetic Monitoring][8]. If this header is present, the value is expected to be one of: `rum`, `synthetics`, `synthetics-browser`.
```
Comment (zacharycmontoya) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:606]: @brett0000FF The use of the word `the Datadog service` doesn't feel right in this instance because the accepted values here are either `rum`, `synthetics`, `synthetics-browser`. Can we revert this change and use the word `product` or is there another term that you think would be more suitable?

Comment (brett0000FF) [reply content/en/tracing/trace_collection/trace_context_propagation/_index.md:606]: Gotcha, that works for me. I mainly changed this because one of our Vale rules checks for "Datadog product" and recommends "Datadog service" or "Datadog". In this context, I think it's fine to leave!

**Review by brett0000FF — APPROVED** [review]
Looks great! 🚀
